benchmarks/shared/src/main/scala/higherkindness/mu/rpc/benchmarks/AvroBenchmark.scala

```diff
@@ -18,52 +18,52 @@ package higherkindness.mu.rpc.benchmarks
 
 import java.util.concurrent.TimeUnit
 
-import cats.effect.{IO, Resource}
-import higherkindness.mu.rpc.benchmarks.shared.Runtime
+import cats.effect.IO
+import higherkindness.mu.rpc.ChannelForAddress
 import higherkindness.mu.rpc.benchmarks.shared.Utils._
 import higherkindness.mu.rpc.benchmarks.shared.models._
 import higherkindness.mu.rpc.benchmarks.shared.protocols.PersonServiceAvro
-import higherkindness.mu.rpc.benchmarks.shared.server._
+import higherkindness.mu.rpc.benchmarks.shared._
 import higherkindness.mu.rpc.protocol.Empty
-import higherkindness.mu.rpc.testing.servers.ServerChannel
 import org.openjdk.jmh.annotations._
 
 @State(Scope.Thread)
 @BenchmarkMode(Array(Mode.Throughput))
 @OutputTimeUnit(TimeUnit.SECONDS)
-class AvroBenchmark extends Runtime {
+class AvroBenchmark extends ServerRuntime {
 
-  implicit val handler: AvroHandler[IO] = new AvroHandler[IO]
+  @Setup
+  def setup(): Unit = startServer.unsafeRunSync
 
-  def clientIO: Resource[IO, PersonServiceAvro[IO]] =
-    Resource
-      .liftF(PersonServiceAvro.bindService[IO])
-      .flatMap(ServerChannel[IO](_))
-      .flatMap(sc => PersonServiceAvro.clientFromChannel[IO](IO(sc.channel)))
+  def clientCall[B](f: PersonServiceAvro[IO] => IO[B]): B =
+    PersonServiceAvro
+      .client[IO](ChannelForAddress("localhost", grpcPort))
+      .use(f)
+      .unsafeRunTimed(defaultTimeOut)
+      .get
 
   @TearDown
   def shutdown(): Unit = {}
```

Comment:
Shouldn't we shut the server down here, as it's now a "real" server bound to a port?

Reply:
Fixed in 964cde1